new_project/examples/separate_apps_testnet/README.md

```diff
@@ -0,0 +1,219 @@
+# Roadmap to testnet
+
+## Intro
+
+Synced testnet nodes are a requirement for running the COMIT Node Daemon (cnd) against testnet. 
+In order to make your life easier we provide fully synced testnet nodes for your cnd here:
+
+Note that our testnet nodes might not always be running, but are just started before workshops/events.
+If you have any questions please contact us through the [COMIT community chat on gitter](https://gitter.im/comit-network/community).
+
+This guide assumes that you run both the taker and the maker on the same machine (for the example's sake).
+In a more real world scenario maker and taker would of course be separate people on separate machines.
+
+## Setup
+
+### Run cnd taker
+
+Create a config file for the taker called `taker_config.toml` with the following content: 
+
+```toml
+[network]
+listen = ["/ip4/0.0.0.0/tcp/9939"]
+
+[bitcoin]
+network = "testnet"
+node_url = "http://bobtimus.coblox.tech:18332"
+
+[ethereum]
+node_url = "http://bobtimus.coblox.tech:8545"
+
+[http_api.socket]
+address = "0.0.0.0"
+port = 8000
+
+[http_api.cors]
+allowed_origins = "all"
+
+[Data]
+dir = "~/comit/taker"
+```
+
+Create a seed file for the taker called `taker_seed.pem`:
+
+```
+-----BEGIN SEED-----
+ZaBZdOwqLWHTLe9HXEc//wVGdt4VzVkDB/jB7tzkf/c=
+-----END SEED-----
+```
+
+Startup command:
+```bash
+cnd --config taker_config.toml --seed-file taker_seed.pem
+```
+
+### Run cnd maker
+Create a config file for the taker called `maker_config.toml` with the following content: 
+
+```toml
+[network]
+listen = ["/ip4/0.0.0.0/tcp/19939"]
+
+[bitcoin]
+network = "testnet"
+node_url = "http://bobtimus.coblox.tech:18332"
+
+[ethereum]
+node_url = "http://bobtimus.coblox.tech:8545"
+
+[http_api.socket]
+address = "0.0.0.0"
+port = 18000
+
+[http_api.cors]
+allowed_origins = "all"
+
+[Data]
+dir = "~/comit/maker"
+```
+
+Create a seed file for the taker called `maker_seed.pem`:
+```
+-----BEGIN SEED-----
+AaBZdOwqLWHTLe9HXEc//wVGdt4VzVkDB/jB7tzkf/c=
+-----END SEED-----
+```
+
+Startup command:
+```
+cnd --config maker_config.toml --seed-file maker_seed.pem
+```
+
+### Key/account generation and funding for both wallets
+
+Keep in mind that when running both maker and taker on the same machine you have to create a Bitcoin and Ethereum account for both maker and taker, thus you have to create and fund **two Bitcoin** and **two Ethereum** accounts.
+
+Configuration in `env` file (assuming that you run on both maker and taker from the same machine):
+
+```
+BITCOIN_HD_KEY_TAKER=EXTENDED_PRIVATE_KEY_GENERATED_BY_WAGYU
```

Comment:
**If** we want to use the way of providing an env file, we could just provide a script that assumes wagyu and generates an env file for the user instead of documenting the steps here.